controller/helper/process-survey-instances.js

```diff
@@ -12,17 +12,41 @@ const moment = require('moment');
  * @returns {Object} Complience chart data
  */
 function processSurveyInstances (surveys) {
+    const filterSurveyByState = surveys.filter((survey) => {
+        return survey.state === 'completed';
+    });
+
     return {
         // Using UTC as the date gets modified to the local time (GMT in this case) when UTC not used.
-        labels: surveys.map((survey) => {
-            return moment(survey.startTime).utc().format('MM/DD/YYYY HH:mm');
-        }),
-        datasets: pickTimeLeft(surveys)
+        labels: pickDates(filterSurveyByState),
+        datasets: pickTimeLeft(filterSurveyByState)
     };
 }
 
 /**
  * Takes in a Survey Instances and get the % time left to be shown on complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of datetimes
+ */
+function pickDates (surveys) {
+    const dates = surveys.map((survey) => {
+        return moment(new Date(survey.startTime).toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    });
+
+    const lastDate = surveys.map((survey) => {
+        return moment(new Date(survey.dateCompleted).toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    });
+
+    const missingValue = -1;
+
+    if (dates.indexOf(lastDate) <= missingValue) {
+        dates.push(lastDate);
+    }
+    return dates;
```

Comment:
@ChristianMurphy This is my mistake. Manohara has done it right. 
lastDate represent the trial end date for particular patient (for example in SCD 5 week) as survey instance may not always represent trial duration.
This single date value should be appended to all the start Date of survey instances per patient to show aggregated view on compliance chart.
I need to change it to single lastDate value.